app/views/refinery/blog/posts/index.rss.builder

```diff
@@ -1,16 +1,22 @@
 xml.instruct! :xml, :version => "1.0"
-xml.rss :version => "2.0" do
+xml.rss version: "2.0", 'xmlns:atom': 'http://www.w3.org/2005/Atom', 'xmlns:media': 'http://search.yahoo.com/mrss/' do
   xml.channel do
     xml.title Refinery::Core.site_name
     xml.description Refinery::Core.site_name + " Blog Posts"
     xml.link refinery.blog_root_url
+    xml.tag! 'atom:link', href: refinery.blog_rss_feed_url, rel: 'self', type: 'application/rss+xml'
 
     @posts.each do |post|
       xml.item do
         xml.title post.title
-        xml.description post.body
+        xml.description blog_post_teaser(post)
+        xml.category post.categories.map {|x| x.title }.join('/') if  post.categories.length > 0
         xml.pubDate post.published_at.to_s(:rfc822)
         xml.link refinery.blog_post_url(post)
+        xml.guid refinery.blog_post_url(post)
+        post.images.each do |image|
```

Comment:
Blog posts don't have an images association?

Reply:
It is added by refinerycms-page-images...
I guess we can either
* remove image support from the RSS
* add some check if `images` is available

Any preference on this matter?

Reply:
I'm happy to just add some check if it's available as to me it seems useful and a somewhat expected use-case